services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -138,6 +138,25 @@ export const EnvironmentListItem: React.FC<EnvironmentListItemProps> = ({
         }
         return otherRelease?.sourceCommitId + ': ' + otherRelease?.sourceMessage;
     };
+    const getDeploymentMetadata = (): string => {
+        if (!application) {
+            return ``;
+        }
+        if (!(application && application.version === release.version)) {
+            return '';
+        }
+        if (application.deploymentMetaData === null) {
+            return '';
+        }
+        const deployedBy = application.deploymentMetaData?.deployAuthor ?? 'unknown';
+        const deployedUNIX = application.deploymentMetaData?.deployTime ?? '';
+        if (deployedUNIX === '') {
+            return 'Deployed by ' + deployedBy;
+        }
+        const deployedDate = new Date(+deployedUNIX * 1000);
+        const returnString = 'Deployed by ' + deployedBy + ' on ' + deployedDate;
```

Comment:
For displaying dates we have a special component here: https://github.com/freiheit-com/kuberpult/blob/main/services/frontend-service/src/ui/components/FormattedDate/FormattedDate.tsx#L52C8-L52C8